src/core/Geometry.js

```diff
@@ -196,21 +196,21 @@ Geometry.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
 
 		}
 
-		var positions = attributes.position.array;
-		var normals = attributes.normal !== undefined ? attributes.normal.array : undefined;
-		var colors = attributes.color !== undefined ? attributes.color.array : undefined;
-		var uvs = attributes.uv !== undefined ? attributes.uv.array : undefined;
-		var uvs2 = attributes.uv2 !== undefined ? attributes.uv2.array : undefined;
+		var positions = attributes.position;
```

Comment:
I would be in favor of using `position` for the attribute itself:
```js
var position = attributes.position;
```
In the case of the array, we use the plural:
```js
var positions = attributes.position.array;
```
JMHO. :-)

Reply:
Done.